website/blog/2018-03-20-our-reacticon.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Our Reacticon
+author: Adrien, Julien and Pierre
+authorURL: https://twitter.com/Front_Commerce
+---
+
+A few days ago we attended [Reacticon](http://reacticon.org/), the first
+conference about <abbr title="Progressive Web Apps">PWA</abbr> in the Magento
+ecosystem.
+
+It was composed of one day of conferences and another one of code! Here are our
+key takeaways.
+
+<!--truncate-->
+
+## A friendly community
+
+First of all, we want to thank every attendees, speakers and of course
+organizers. It was a wonderful event thanks to you.
+
+Everyone was keen to share insights and answer questions, which led to very
+interesting discussions.
+
+## <abbr title="Progressive Web App">PWA</abbr> as a buzzword
+
+<abbr title="Progressive Web App">PWA</abbr> was the keyword of this event.
+Everyone is excited that they will soon be able to create wonderful eCommerce
+experience with Magento. But, it seems that there is still a lot a confusion
+around it.
+
+One feel that they need to use React and create a
+<abbr title="Single Page Application">SPA</abbr> to create a
+<abbr title="Progressive Web App">PWA</abbr>. Another say that you need to offer
+a full offline experience. Some claim that you only need a shortcut to
+your homescreen.
+
+Fortunately, [Shane Osbourne](https://twitter.com/shaneOsbourne) was here to
+remind everyone to improve their user and developer experience step by step.
+Focus on one feature at a time, and in the long run, you'll
+deliver awesome shopping experience to your customers.
+
+## Magento Research is promising
+
+[James Zetlen](https://twitter.com/jameszetlen) gave more information about
+<abbr title="Progressive Web App">PWA</abbr> Studio, and the processes behind
+it. He introduced the
+[Magento Research Github organization](https://github.com/magento-research), a
+new way for Magento to innovate and interact from the community. We could
+expect:
+
+* open source code shared early with the community to get feedbacks
+* more focused tools (reusable outside Magento)
```

Comment:
the meaning of 'focused' is not obvious to me in this context, what were you trying to say specifically ?

If the reusability is to be emphasized I would have simply said 'flexible'